rolling/distribution.yaml

```diff
@@ -358,6 +358,16 @@ repositories:
       url: https://github.com/AprilRobotics/apriltag.git
       version: master
     status: maintained
+  apriltag_detector:
```

Comment:
Note that your next step here will be to get an entry in the ros2-gbp org: https://github.com/ros2-gbp/ros2-gbp-github-org